src/components/Dropdown/Dropdown.tsx

```diff
@@ -147,8 +147,8 @@ export class Dropdown extends BaseComponent<IDropdownProps, any> {
   }
 
   private _renderDropdownOption(item: IDropdownOption): JSX.Element {
-    let defaultRenderer = (item: IDropdownOption): JSX.Element => {
-      return <span>{item.text}</span>;
+    let defaultRenderer = (defaultItem: IDropdownOption): JSX.Element => {
```

Comment:
When this code executes, it will create a new closured function for every item. A more efficient approach is this:

```
public render() {
  let { onRenderItem = this._onRenderItem } = this.props;

  render (
     items.map(item => (
        <li key={item.key}>
             { onRenderItem(item, this._onRenderItem) }
        </li>
}

@autobind
private _onRenderItem(item: IDropdownOption): JSX.Element) {
  return <span>{item.text}</span>
}